qCC/ccCommandLineCommands.cpp

```diff
@@ -5875,7 +5905,19 @@ bool CommandSFOperation::process(ccCommandLineInterface& cmd)
 		sf2.isConstantValue = true;
 		sf2.constantValue = value;
 	}
-	
+
+	//in place modifier, to keep old commands intact we should keep it in place by default. However it makes the command line inconsistence, because the SF_ARITHMETIC works the other way.
+	bool inPlace = true;
+	if (!cmd.arguments().empty())
+	{
+		if (cmd.IsCommand(cmd.arguments().front(),COMMAND_SF_OP_NOT_IN_PLACE))
```

Comment:
I would have placed it first (even if optional)

Reply:
i'll do a bigger change and replace the parameter loading to a while loop similar to other commands. So it can be placed anywhere.

Reply:
On second thought it is not gonna work in this case. I'll put this option to the front as optional.